test/cpp/jit/CMakeLists.txt

```diff
@@ -115,6 +115,11 @@ add_executable(test_jit
   ${JIT_TEST_SRCS}
 )
 
+if(USE_ASAN)
```

Comment:
can you document here why UBSan is enabled when ASan is requested? Also, can you update pytorch/CMakeLists.txt to indicate that we also enable UBSan under jit with the USE_ASAN setting.